src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -1,15 +1,173 @@
 import { ArrowForwardIcon } from '@chakra-ui/icons'
-import { Button, HStack, Stack } from '@chakra-ui/react'
+import {
+  Box,
+  Button,
+  Flex,
+  HStack,
+  Skeleton,
+  SkeletonCircle,
+  Tag,
+  TagLabel
+} from '@chakra-ui/react'
+import { CAIP19 } from '@shapeshiftoss/caip'
+import { AprTag } from 'plugins/cosmos/components/AprTag/AprTag'
+import { StakingAction } from 'plugins/cosmos/components/modals/Staking/Staking'
+import { useMemo } from 'react'
 import { NavLink } from 'react-router-dom'
+import { Column } from 'react-table'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
 import { Card } from 'components/Card/Card'
-import { Text } from 'components/Text'
+import { ReactTable } from 'components/ReactTable/ReactTable'
+import { RawText, Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+import { BigNumber, bnOrZero } from 'lib/bignumber/bignumber'
+import { selectAssetByCAIP19 } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
 
-import { StakingOpportunitiesRow } from './StakingOpportunitiesRow'
+type StakingOpportunity = {
+  id: number
+  moniker: string
+  apr: BigNumber
+  cryptoAmount?: BigNumber
+  rewards: Rewards
+}
+
+type Rewards = {
+  fiatRate: BigNumber
+  stakedRewards?: BigNumber
+}
+
+type StakingOpportunitiesProps = {
+  assetId: CAIP19
+  opportunities: StakingOpportunity[]
+}
+
+type ValidatorNameProps = {
+  moniker: string
+  isStaking: boolean
+}
+
+export const ValidatorName = ({ moniker, isStaking }: ValidatorNameProps) => {
+  const isLoaded = true
+  const assetIcon = isStaking
+    ? 'https://assets.coincap.io/assets/icons/eth@2x.png'
+    : 'https://assets.coingecko.com/coins/images/16724/small/osmo.png?1632763885'
 
-export const StakingOpportunities = () => {
-  // TODO: wire up with real validator data
-  const validators = [{ id: 1, name: 'Cosmos Validator' }]
+  return (
+    <Box cursor='pointer'>
+      <Flex alignItems='center' maxWidth='180px' mr={'-20px'}>
+        <SkeletonCircle boxSize='8' isLoaded={isLoaded} mr={4}>
+          <AssetIcon src={assetIcon} boxSize='8' />
+        </SkeletonCircle>
+        <Skeleton isLoaded={isLoaded} cursor='pointer'>
+          {isStaking && (
+            <Tag colorScheme='blue'>
+              <TagLabel>{moniker}</TagLabel>
+            </Tag>
+          )}
+          {!isStaking && <RawText fontWeight='bold'>{`${moniker}`}</RawText>}
+        </Skeleton>
+      </Flex>
+    </Box>
+  )
+}
+
+export const StakingOpportunities = ({ assetId, opportunities }: StakingOpportunitiesProps) => {
+  const isLoaded = true
+  const isStaking = opportunities.some(x => x.cryptoAmount)
+  const assetSymbol = useAppSelector(state => selectAssetByCAIP19(state, assetId)).symbol
 
+  const { cosmosGetStarted, cosmosStaking } = useModal()
+
+  const handleGetStartedClick = (e: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
I believe it's best practice to explicitly import `React`, as not doing so can lead to runtime errors.

`import React from 'react'`.